bin/test

```diff
@@ -152,11 +151,7 @@ function runTerraform() {
 
   dockerCompose up -d terraform
 
-  terraformRun <<EOF
-    terraform init $target_dir/
-    terraform plan $target_dir/
-    terraform apply -auto-approve $target_dir/
-EOF
+  terraformRun "terraform init $target_dir/ && terraform plan $target_dir/ && terraform apply -auto-approve $target_dir/"
```

Comment:
can we still have line breaking like before? it's easier to read

Reply:
I've added line breaks.  It's not _as_ nice as before, but still reasonable.